packages/app/src/Feed/EventPublisher.ts

```diff
@@ -251,18 +260,6 @@ export default function useEventPublisher() {
         return await signEvent(ev);
       }
     },
-    saveRelays: async () => {
-      if (pubKey) {
-        const ev = NEvent.ForPubKey(pubKey);
-        ev.Kind = EventKind.ContactList;
-        ev.Content = JSON.stringify(relays);
-        for (const pk of follows) {
-          ev.Tags.push(new Tag(["p", pk], ev.Tags.length));
-        }
-
-        return await signEvent(ev);
-      }
-    },
```

Comment:
I don't think we're supposed to do this. At least from reading [nip-02](https://github.com/nostr-protocol/nips/blob/master/02.md) and [nip-65](https://github.com/nostr-protocol/nips/blob/master/65.md) it doesn't seem like we're supposed to do this. Let me know if I'm misunderstanding something.

Reply:
Yes we are supposed to do this, relays are stored in the contact list ofc!

Reply:
Doh. I couldn't find it in the docs. Where is this documented?

Reply:
Nip-02 says the value of `content` should be ignored.

Reply:
Its not documented but this is actually where all clients store relays right now, NIP-65 is new

Reply:
Oh maybe that's why a lot of relays don't seem to support nip-65. Let me revert my changes and update to render the relays pulled from the data from the kind 3 event rather than kind 10002 event then. I think that should fix the stale relays bug.